package.json

```diff
@@ -29,12 +29,12 @@
     "format": "prettier --write '**'",
     "format:check": "prettier --check '**'",
     "lint": "eslint src test ./*.js",
-    "lint:check": "eslint --print-config ./aw.config.js",
+    "lint:check": "eslint src test ./*js",
```

Comment:
This seem to be the same as the `lint` script. Appart from it having broken regex. Just remove this script?